docker-compose-3pc-test.yml

```diff
@@ -2,101 +2,20 @@ version: '3'
 
```

Comment:
The services present in docker-compose-3pc do not need to be in here in hindsight. It saves redunancy in these yaml files. You can specify multiple files when running docker compose. I.e., standing up only the system without loadgen would be 

```
docker compose -f docker-compose-3pc.yml up -d
```

and if you want the loadgen too you'd do

```
docker compose -f docker-compose-3pc.yml -f docker-compose-3pc-test.yml up -d
```